pandas/core/indexes/multi.py

```diff
@@ -946,7 +946,9 @@ def _format_native_types(self, na_rep='nan', **kwargs):
             new_codes.append(level_codes)
 
         if len(new_levels) == 1:
-            return Index(new_levels[0])._format_native_types()
+            # a single-level multi-index
+            return Index(new_levels[0].take(new_codes[0])).\
```

Comment:
Shouldn't need the explicit line break here

Reply:
Reformatted